src/material/schematics/ng-update/migrations/hammer-gestures-v9/hammer-template-check.ts

```diff
@@ -87,7 +87,7 @@ export function isHammerJsUsedInTemplate(html: string): {
       }
     });
   };
-  visitNodes(document.childNodes);
+  visitNodes((document as any).childNodes as parse5.ChildNode[]);
```

Comment:
Is there a reason the type is unspecified?

Reply:
I swapped to the alternative: `visitNodes((document as unknown as parse5.Document).childNodes);`

Reply:
Oh, I was getting typescript errors about it complaining that `document` was of type `unknown`

Reply:
I think the exact error was:
`ts2571: object is of type 'unknown'`